example-nrlp-wire-format.md

```diff
@@ -0,0 +1,194 @@
+# Examples of Wire Format Options
+
+## One Throughput Instance
+
+Let's consider the example of this advice:
+
+~~~
+{
+    "throughput-advice": [
+        {
+            "direction": 0,
+            "scope": false,
+            "tc": 0,
+            "cir": 50,
+            "cbs": 10000
+        }
+    ]
+}
+~~~
+
+### DHCP
+
+This corresponds to the following NRLP DHCP option:
+
+~~~
+ 0                   1
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|    0xE0       |     0x0C      |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|     0x00      |     0x0A      |
```

Comment:
Can you add comments explaining the bits?  I don't see immediate association between the JSON and those hex values.

Reply:
Added those. Hope this is better.
This encoding still assumes we are using one single bit for direction (but will need to change that to two bits for compactness with the same policy applies for both directions).